src/prashant/BankSystem.java

```diff
@@ -0,0 +1,107 @@
+/*Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+
+package prashant;
+
+public class BankSystem {
+
+	int CreditAmt;
+	int DebitAmt;
+	String Cname;
+	int Totalbalance = 10000;
+	int bal;
+	int CreditIteration;
+	int DebitIteration;
+
+	static int totaldebitcount = 0;
+	static int totalcreditcount = 0;
+	static int totalbalcount = 0;
+
+	// Method for Credit amount calculation
+
+	void Credit(String CustName, int Cramt) {
+		Cname = CustName;
+		CreditAmt = Cramt;
+		System.out.println("Credit Amount  :" + Cramt);
+		Totalbalance = Totalbalance + CreditAmt;
+		CreditIteration++;
+		totalcreditcount++;
+	}
+
+	// Method for Debit amount calculation
+
+	void Debit(String CustName, int Dramt) {
```

Comment:
You should check if (totalBalance >= requestedBalance ) then only you should debit amt from main balance